.gitignore

```diff
@@ -1,3 +1,38 @@
 /VATSpyConfig.xml
 /WorldMap.dat
 /Meta.json
+QGIS/vattech.ico
+QGIS/RUS.dat
+QGIS/GCCO.geojson
+QGIS/main.exe
+QGIS/config.json
+QGIS/GCCO.cpg
+QGIS/GCCO.dat
+QGIS/GCCO.dbf
+QGIS/GCCO.prj
+QGIS/GCCO.qgz
+QGIS/GCCO.qmd
+QGIS/GCCO.shp
+QGIS/GCCO.shx
+QGIS/LPPC.dat
+QGIS/LPPC.geojson
+QGIS/LPPC.qmd
+QGIS/polygons.dat
+QGIS/POLYGONS.geojson
+QGIS/POLYGONS.qmd
+QGIS/URAL.cpg
+QGIS/URAL.dat
+QGIS/URAL.dbf
+QGIS/URAL.geojson
+QGIS/URAL.prj
+QGIS/URAL.qmd
+QGIS/URAL.shp
+QGIS/URAL.shx
+QGIS/lppcgcco.cpg
+QGIS/lppcgcco.dat
+QGIS/lppcgcco.dbf
+QGIS/LPPCGCCO.geojson
+QGIS/lppcgcco.prj
+QGIS/lppcgcco.qmd
+QGIS/lppcgcco.shp
+QGIS/lppcgcco.shx
```

Comment:
I assume this was not meant to be added?

Reply:
Yes, sorry, forgot to uncheck this when pushing the PR. How do I go about removing this change from the PR, not too familiar with Git :P.

Reply:
![image](https://user-images.githubusercontent.com/42404718/135592564-d814131a-0c46-41d6-a495-e876ee14ef21.png)